### PR TITLE
update docs for supported JDBC parameters

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: docs/connecting.txt [hunk 1]
```diff
@@ -89,29 +89,17 @@ connection with the database superuser ``crate``.
...
+In addition, connection parameters can be passed via the JDBC URL:
 
 .. code-block:: java
 
     Connection conn = DriverManager.getConnection("crate://localhost:5432/?property1=value1&property2=value2");
 
-CrateDB JDBC driver supports following properties:
+.. NOTE::
 
-:strict:
```
Comment (chaudum) [inline docs/connecting.txt]: The `strict` parameter does still exist.

File: docs/connecting.txt [hunk 2]
```diff
@@ -89,29 +89,17 @@ connection with the database superuser ``crate``.
...
-    **Default**: ``false``.
+   The CrateDB JDBC driver supports all of `the PostgreSQL JDBC driver
+   parameters`_, though not all of them will make sense for CrateDB.
```
Comment (chaudum) [inline docs/connecting.txt]: I think we should come up with a list of parameters that we think are relevant and have effect to the connection with CrateDB.

Comment (chaudum) [reply docs/connecting.txt]: Relevant properties provided from the pgjdbc upstream are:
* user
* password
* ssl
* loglevel
* prepareThreshold
* loginTimeout
* connectTimeout
* socketTimeout
* tcpKeepAlive
* readOnly
* currentSchema
* loadBalanceHosts

Comment (nslater) [reply docs/connecting.txt]: @chaudum should we just list these and link through to the parent documentation, or should we duplicate the parent documentation? or something crate specific?

I am concerned that the documentation for `strict` is so verbose. it is going to make a list of either non-documented or minimally documented parameters look out-of-place

Comment (chaudum) [reply docs/connecting.txt]: I would duplicate the description of relevant parameters and additionally add the link to pgjdbc (full reference).

`strict` is a kind of an important parameter, it's ok if the description is verbose.

File: docs/connecting.txt [hunk 3]
```diff
@@ -89,13 +89,15 @@ connection with the database superuser ``crate``.
...
+In addition, connection parameters can be passed via the JDBC URL:
 
 .. code-block:: java
 
     Connection conn = DriverManager.getConnection("crate://localhost:5432/?property1=value1&property2=value2");
 
-CrateDB JDBC driver supports following properties:
+CrateDB JDBC driver supports all of `the PostgreSQL JDBC driver parameters`_,
+though not all of them will make sense for CrateDB. We have listed the most
+important ones here.
```
Comment (chaudum) [inline docs/connecting.txt]: where?

Comment (nslater) [reply docs/connecting.txt]: this was just a partial fixup so I could switch branches :)

**Comment by nslater** [comment]
@chaudum could you take another look at this now please?

File: docs/connecting.txt [hunk 4]
```diff
@@ -114,4 +115,95 @@ CrateDB JDBC driver supports following properties:
...
+
+    The database user on whose behalf the connection is being made.
+
+:password: ``<String>``
+
+    The database user's password.
+
+:ssl: ``true``
+
+    Connect using SSL. The driver must have been compiled with SSL support.
```
Comment (mxm) [inline docs/connecting.txt]: Would remove the second sentence. AFAIK we make sure the driver includes SSL support.

Comment (chaudum) [reply docs/connecting.txt]: Good catch!

**Comment by nslater** [comment]
closing this because we've decided to take a different approach

**Pull request closed without merging**